packages/flutter/lib/src/services/asset_manifest.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../foundation.dart';
```

Comment:
since this is in services it should import through flutter/foundation and not a relative import.